libr/debug/debug.c

```diff
@@ -661,13 +660,47 @@ R_API const char *r_debug_reason_to_string(int type) {
 	case R_DEBUG_REASON_TRAP: return "trap";
 	case R_DEBUG_REASON_SWI: return "software-interrupt";
 	case R_DEBUG_REASON_INT: return "interrupt";
-	case R_DEBUG_REASON_FPU: return "fpu";
+	case R_DEBUG_REASON_FPU: return "Floating point exception";
 	case R_DEBUG_REASON_STEP: return "step";
 	case R_DEBUG_REASON_USERSUSP: return "suspended-by-user";
 	}
 	return "unhandled";
 }
 
+R_API const char *r_signal_to_human(int signum) {
+	switch (signum) {
+		case R_DEBUG_REASON_SIGALRM: return "Alarm clock";
```

Comment:
wrong indent and if those are signals, dont use RDEBUG REASONS.. 

also as i said in the pad, rename R_DEBUG_REASON to R_DEBUG_STOPREASON